src/imgui/csvparser.hpp

```diff
@@ -0,0 +1,188 @@
+#ifndef CSV_PARSER_H
+#define CSV_PARSER_H
+
+#include "fileHandler.hpp"
+#include "error.hpp"
+
+#include <string>
+#include <tl/expected.hpp>
+
+namespace CSV {
+
+using Row = std::vector<std::string>;
+using Error = cash_overflow::error::Error;
+
+namespace MISC {
```

Comment:
is this supposed to be "hidden" ? usually ppl use a `detail` namespace for it

Reply:
done